tests/test_noninteractive_mode.py

```diff
@@ -6,7 +6,9 @@
 import pytest
 from mssqltestutils import create_mssql_cli
 
-_FILE_ROOT = '%s/test_query_results/' % os.path.dirname(os.path.abspath(__file__))
+_SLASH = '\\' if os.name == 'nt' else '/'
```

Comment:
I think you can use os.path.join() to create file paths without having use the underlying platform specific path separator.